### PR TITLE
Removing files from other env which not exists in current env

### PR DESCRIPTION
**Pull request by DeryabinSergey**
Looks like #215 

Remove files from other environments not exists in current environment

| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Fixed issues  | #215

### CODE REVIEW

File: init [hunk 1]
```diff
@@ -78,6 +78,30 @@ foreach ($files as $file) {
     }
 }
 
+$filesToRemove = [];
+$skipFiles = isset($env['skipFiles']) && $env['skipFiles'] ? $env['skipFiles'] : [];
```
Comment (samdark) [inline init:82]:
```suggestion
$skipFiles = !empty($env['skipFiles']) ? $env['skipFiles'] : [];
```

**Review by samdark — APPROVED** [review]
Looks alright to me.

File: init (hunk 1 above)
Comment (DeryabinSergey) [reply init]: Good idea, thanks a lot

File: init [hunk 2]
```diff
@@ -78,6 +78,30 @@ foreach ($files as $file) {
     }
 }
 
+$filesToRemove = [];
+$skipFiles = !empty($env['skipFiles']) ? $env['skipFiles'] : [];
+foreach(array_column($envs, 'path') as $envPath) {
+    if ($env['path'] == $envPath) continue;
```
Comment (bizley) [inline init:84]:
```suggestion
    if ($env['path'] === $envPath) continue;
```

**Review by bizley — COMMENTED** [review]
Please use the [Yii coding style](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md) (indentations, etc.)

**Comment by samdark** [comment]
Thanks!

**Pull request merged**